cfme/tests/test_manual.py

```diff
@@ -3358,20 +3284,6 @@ def test_host_tagged_crosshair_op_vsphere55():
     pass
 
 
-@pytest.mark.manual
-@test_requirements.reconfigure
-@pytest.mark.tier(1)
-def test_vm_reconfig_add_remove_network_adapters_vsphere67_nested_vmkernel():
```

Comment:
why did you remove this test as well as other 3 `test_vm_reconfig_add_remove_network_adapters_vsphere67_nested*`? Or was it renamed?

Reply:
Yes I renamed it --> ```def test_vm_reconfig_add_remove_network_adapters(adapters_type):```